src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -142,6 +164,10 @@ function useChatUtilsHook(): UseChatUtilsHook {
     });
   };
 
+  const goToContactScreen = () => {
+    navigation.navigate('ContactScreen');
+  };
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new function `goToContactScreen` has been added which navigates to the 'ContactScreen'. This addition seems fine as it provides a way to navigate to the contact screen.



<!-- This is an auto-generated comment by OSS CodeRabbit -->